README.md

```diff
@@ -14,44 +14,41 @@ SPDX-License-Identifier: MIT
 [![Deployment](https://github.com/StanfordBDHG/ENGAGE-HF-Web-Frontend/actions/workflows/deployment.yml/badge.svg)](https://github.com/StanfordBDHG/ENGAGE-HF-Web-Frontend/actions/workflows/deployment.yml)
 [![codecov](https://codecov.io/gh/StanfordBDHG/ENGAGE-HF-Web-Frontend/graph/badge.svg?token=PsKyNz7Woe)](https://codecov.io/gh/StanfordBDHG/ENGAGE-HF-Web-Frontend)
 
-## How To Use ENGAGE-HF Web Frontend
+Web Frontend for the ENGAGE-HF project.
 
-The ENGAGE-HF Web Frontend repository contains a Next.js project providing automated GitHub Actions and setups for code linting, testing & test coverage reports, docker deployments, a docker compose setup, local packages for modular deployment.
+## Behavior
+
+This repository provides dashboard to manage ENGAGE-HF application, for admins and clinicians. Clinicians can interact with patients, adjust their recommended medications, allergies and appointments.
+
+![Application](resources/appScreenshot.png)
+
+## Stack
+
+The ENGAGE-HF Web Frontend repository contains a React application built with TypeScript, Vite and Firebase. It provides automated GitHub Actions, setups for code linting, testing & test coverage reports, docker deployments, a docker compose setup.
+
+In order to run Web Frontend, you need to use actual Firebase environment or Emulator with seeded data. For developing locally, it's best to use the Emulator. Refer to ["Using the emulator for client applications" section of ENGAGE-HF-Firebase](https://github.com/StanfordBDHG/ENGAGE-HF-Firebase?tab=readme-ov-file#using-the-emulator-for-client-applications) for an instruction.
 
 ## Getting Started
 
-You can run the project using the following command. You will need to install Node.js and npm, e.g., using [homebrew (recommended for macOS)](https://formulae.brew.sh/formula/node) or the official [Node.js installer](https://nodejs.org/en/download).
+This project uses Node.js v22. Install Node.js, e.g. using [nvm](https://github.com/nvm-sh/nvm?tab=readme-ov-file#installing-and-updating), [homebrew (for macOS)](https://formulae.brew.sh/formula/node) or official [Node.js installer](https://nodejs.org/en/download) .
 
-1. Install All Dependencies
+1. Install all dependencies
 
 ```bash
 npm install
 ```
 
-2. Setup Firebase Environment Values
+2. Setup environment variables
 
-Create an `.env.local` file or inject the following environment variables required for the Google Firebase Setup:
-
-```
-VITE_PUBLIC_FIREBASE_API_KEY=
-VITE_PUBLIC_FIREBASE_AUTH_DOMAIN=
-VITE_PUBLIC_FIREBASE_PROJECT_ID=
-VITE_PUBLIC_FIREBASE_STORAGE_BUCKET=
-VITE_PUBLIC_FIREBASE_MESSAGING_SENDER_ID=
-VITE_PUBLIC_FIREBASE_APP_ID=
-VITE_PUBLIC_EMULATOR=
-VITE_PUBLIC_EMAIL_PASSWORD_SIGN_IN=""
-```
+Refer [.env.example](.env.example) file for environment variables documentation. Copy `.env.example` to `.env.local` and adjust if necessary.
```

Comment:
Might be good to document that they would need to adjust if they need to do anything; and maybe even an explanation to all these values in a table and/or a reference to the Firebase documentation?
```suggestion
Refer to [.env.example](.env.example) file for environment variables documentation. Copy `.env.example` to `.env.local` and adjust if necessary.
```

Reply:
Oh, perfect; thank you! Overlooked that 👍